### ISSUE TITLE
Glitchy hero image on /new

### ISSUE THREAD

**Issue by amychurchwell** [mozilla/bedrock#9312]
## Description

While testing in browserstack, I noticed in IE11 on Windows 10 the hero image animation causing a glitchy effect.
Others could only reproduce the bug in Windows 8.1. We should do a static image fallback for all IE's.

![Aug-14-2020 09-04-15](https://user-images.githubusercontent.com/30483988/90269222-2bdbfd00-de0d-11ea-981c-6640b8df3dc0.gif)

**Comment by paras151** [comment]
Btw, it is not easy to download internet explorer now it has been disabled through a microsoft edge update on windows 10 and is not supported in windows 8.1 or 7, as mentioned [here](https://support.microsoft.com/en-us/windows/internet-explorer-downloads-d49e1f0d-571c-9a7b-d97e-be248806ca70).

**Comment by alexgibson** [comment]
Agreed, I think we can close this as wontfix. This bug has been open a long time, and even Firefox is no longer supported on Windows 8.1 and below.
